cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -50,9 +50,9 @@ current-context: service-account-context
 EOF
   local -r kubeconfig_base64=$(echo "${kubeconfig}" | base64 -w0)
   read -r -d '' secretyaml <<EOF
-apiVersion: v1beta1
+apiVersion: v1beta3
 kind: Secret 
-id: token-${safe_username}
+name: token-${safe_username}
```

Comment:
Sadly, secrets.md is wrong:
https://github.com/GoogleCloudPlatform/kubernetes/blame/master/docs/secrets.md

Please fix that, also.

Take a look at the conversion tips:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips

Let me know what's not clear.

You can also look at the source:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L1718

cc @caesarxuchao @nikhiljindal